src/Illuminate/Collections/Traits/EnumeratesValues.php

```diff
@@ -321,6 +321,22 @@ public function firstWhere($key, $operator = null, $value = null)
         return $this->first($this->operatorForWhere(...func_get_args()));
     }
 
+    /**
+     * Get a single key's value from the first matching item in the collection.
+     *
+     * @param  string  $key
+     * @param  mixed  $default
+     * @return mixed
+     */
+    public function value($key, $default = null)
+    {
+        if ($value = $this->firstWhere($key)) {
+            return data_get($value, $key, $default);
+        }
+
+        return $default;
```

Comment:
Is there a reason you didn't use `value($default)` here?

Reply:
Nope, I just didn’t think of it 😄